docs/managing-secrets-with-vault.md

```diff
@@ -22,38 +22,210 @@ about deploying Vault on Kubernetes for production, see [Vault on Kubernetes
 deployment
 guide](https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-raft-deployment-guide).
 
-_Prerequisites_
+### Prerequisites
```

Comment:
The structure of the guide matches that of the Red Hat style, so we don't use headers here. Please change this back to italics.